Formula/pachi.rb

```diff
@@ -18,40 +18,28 @@ class Pachi < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "e0fc26989c0cf90b6fa2256e129b0b87993464ec27ad88fefe569abdd9702292"
   end
 
-  resource "patterns" do
-    url "https://sainet-dist.s3.amazonaws.com/pachi_patterns.zip"
-    sha256 "73045eed2a15c5cb54bcdb7e60b106729009fa0a809d388dfd80f26c07ca7cbc"
-  end
+  depends_on "caffe"
```

Comment:
Is `caffe` required? We no longer allow `caffe` to be used as a dependency because it is deprecated.

Reply:
Yes, currently Pachi depends on caffe to evaluate its neural network. It's possible to do a nodcnn build, in which case caffe is not needed, but really unfortunate for users (Pachi loses much of its value without dcnn).

Reply:
I think that's the trade-off we'll have to make here. We can use dcnn once it works with software that is not deprecated.